Mathlib/Data/Set/Pointwise/SMul.lean

```diff
@@ -91,11 +91,10 @@ theorem image2_smul : image2 SMul.smul s t = s • t :=
 #align set.image2_smul Set.image2_smul
 #align set.image2_vadd Set.image2_vadd
 
-@[to_additive add_image_prod]
+-- @[to_additive add_image_prod] -- Porting note: bug in mathlib3
```

Comment:
This inserts a translation to an existing but unrelated declaration? We should probably make this `@[to_additive vadd_image_prod]` if that works